angular-bind-html-compile.js

```diff
@@ -6,15 +6,17 @@
     module.directive('bindHtmlCompile', ['$compile', function ($compile) {
         return {
             restrict: 'A',
-            link: function (scope, element, attrs) {
-                scope.$watch(function () {
+            link: function(scope, element, attrs) {
+                scope.$watch(function() {
                     return scope.$eval(attrs.bindHtmlCompile);
-                }, function (value) {
+                }, function(value) {
```

Comment:
No need to apologize, I'm also a stickler for style in code :). I'll add them in.